class-omnisend-lifterlmsaddon.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Plugin Name: Omnisend for LifterLMS Add-On
+ * Description: A LifterLMS add-on to sync contacts with Omnisend. In collaboration with Omnisend for WooCommerce plugin it enables better customer tracking
+ * Version: 1.0.1
+ * Author: Omnisend
+ * Author URI: https://www.omnisend.com
+ * Developer: Omnisend
+ * Developer URI: https://developers.omnisend.com
```

Comment:
```suggestion
 * Developer URI: https://omnisend.com
```

Reply:
I looked in other Omnisend add oddons on github in all "Developer URI" is https://developers.omnisend.com/. Should i really change it?

Reply:
Changed